docs/en/week02/02-3.md

```diff
@@ -0,0 +1,43 @@
+---
+lang-ref: ch.01-3
+title: Problem Motivation, Linear Algebra, and Visualization
+lecturer: Alfredo Canziani
+authors: Rajashekar Vasantha, Derek Yen, Tony Xu, Ben Stadnick, Prasanthi Gurumurthy
+date: 04 Feb 2021
+---
+
+
+## Resources
+
+Please follow Alfredo Canziani [on Twitter @alfcnz](https://twitter.com/alfcnz). Videos and textbooks with relevant details on linear algebra and singular value decomposition (SVD) can be found by searching Alfredo's Twitter, for example type `linear algebra (from:alfcnz)` in the search box.
+
+
+## [Neural Nets: Rotation and Squashing](https://youtu.be/0TdAmZUMj2k)
+A traditional neural network is an alternating collection of two blocks - the linear blocks and the non-linear blocks. Given below is a block diagram of a traditional neural network. 
+
+<center><img src="{{site.baseurl}}/images/week02/02-3/figure1.png" width="1000px"/></center>
+
+The linear blocks(Rotations, for simplicity) are given by:
```

Comment:
Add a space before `(`, here and everywhere else.

Reply:
Added spaces everywhere